src/Illuminate/Database/Query/Builder.php

```diff
@@ -2729,6 +2730,24 @@ public function insert(array $values)
         );
     }
 
+    /**
+     * Evaluate the values of default attributes passed if closures exist.
+     *
+     * @param  array  $attributes
+     * @return array
+     */
+    private function resolveInsertDefalts(array $attributes)
```

Comment:
Typo here change to `resolveInsertDefaults`

Reply:
Thanks mate. Looks like its gonna be closed so 😉

Reply:
You could send this to `master` 7.0 can not introduce BC

Reply:
It probably will have that "method signature", unless if we are to make a new method.

Reply:
It should not be a problem if you target `master`

Reply:
Another feedback: as a rule of thumb, Laravel uses `protected` and not `private`